LabDrivers/LM500.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jun 7 2019
+For Cryomagnetic LM-500, which I believe is the same machine as CG500.py
```

Comment:
If they are the same, it could be worth sharing the main class like for KT2400 and KT2450